### ISSUE TITLE
Databricks or iPython Notebooks

### ISSUE THREAD

**Issue by capitanou** [databricks/Spark-The-Definitive-Guide#6]
It would be very helpful and time saving to have the databricks or ipython notebooks available to import rather than the .py code files.  Could you upload them separately to the a directory?

**Comment by anabranch** [comment]
You can import those python files into Databricks - you then get notebooks you can follow along with.

Maybe I don't understand your question completely, could you rephrase it so I can try to understand better?

**Comment by capitanou** [comment]
The import option works great for Databricks and appropriately creates a cell for each section of the .py file; however, when working with the .py files in jupyter I can't seem to find a way to import them into the notebook into separate cells.  Using %load will import the entire .py file into a single cell.  

Is there a simple way to import them that I am unaware of?

Perhaps a solution would be to export the code samples to .ipynb which would allow them to be imported in either Databricks or jupyter.

**Comment by anabranch** [comment]
I am not aware of a simple way. Unfortunately, I don't have time to dive into figuring out a solution here. If you find one, please feel free to contribute the changes back.
